.storybook/main.js

```diff
@@ -94,7 +94,7 @@ module.exports = /** @type {Omit<StorybookConfig,'typescript'|'babel'>} */ ({
   },
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31706&commitId=47a6fcfecc7e2f36b3383bfa334c095420f17043&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged.appearance - High Contrast.normal.chromium.png|0| Added|

</details>


  <details><summary><b>Card Converged - Selectable</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance selectable - Subtle.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Subtle.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Subtle.normal.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Subtle.selected.chromium.png|0| Removed|

</details>


  <details><summary><b>SwatchPicker Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.default - RTL.default.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>